learning-bitmark/working-with-bitmarks/issuing-bitmarks.md

```diff
@@ -0,0 +1,183 @@
+
+The Bitmark blockchain is a public global database that stores property rights to non-fungible objects in a token-like container. Because it is a decentralized and politically-neutral system, no single entity has the ability to improperly modify property records or transactions — not even the Bitmark team. 
+
+<br>
+
+>The process of registering Bitmark Certificates for your digital assets bassically contains two steps
+>
+>* Registering your asset - results a *Asset Record* stored on the Bitmark blockchain.
+>
+>* Issuing bitmarks -  results *Issue Records* linking to the corresponding asset record and stored on the Bitmark blockchain.
+> 
+> 
+
+<br>
+
+> **NOTE:** Any user interacting with the Bitmark Property System requires a Bitmark Account.
+> Please refer [Bitmark Account](creating-bitmark-account.md){:target="_blank"} section for instructions of creating a new Bitmark Account.
+
+<br>
+<br>
+
+## Registering Bitmark Certificates using Bitmark App
+
+The Bitmark app registers legal property rights on the public Bitmark blockchain for your digital assets, including personal health and social data, creative works such as art, photography, and music, and other intellectual property. These legal rights determine who owns property and what can be done with it, whether you want to keep it, sell it, or donate it.
+
+<br>
+Here are the steps to register a new property using the Bitmark app:
+
+* On the PROPERTIES screen - Tap **CREATE FIRST PROPERTY** or **+** 
+
+    > It opens the PROPERTIES > REGISTER screen
+
+    <div style="background-color: #efefef; text-align: center;">
+        <img src="images/RegisteringProperties_0.png" alt="Properties screen" title="Properties screen" width="250" style="padding: 20px" />
+        <img src="images/RegisteringProperties_1.png" alt="Properties Register screen" title="Properties Register screen" width="250" style="padding: 20px" />
+    </div>
+
+    <br>
+
+* Tap **PHOTOS** or **FILES** to browse the desired asset
+
+    > It requires permission granted to access `Photos` and/or `Files`
+
+    <div style="background-color: #efefef; text-align: center;">
+        <img src="images/RegisteringProperties_2.png" alt="Grant Permission popup" title="Grant Permission popup" width="250" style="padding: 20px" />
+        <img src="images/RegisteringProperties_3.png" alt="Open Photos action sheet" title="Open Photos action sheet" width="250" style="padding: 20px" />
+        <img src="images/RegisteringProperties_4.png" alt="PHOTOS browsed" title="PHOTOS browsed" width="250" style="padding: 20px" />
+    </div>
+
+    <br>
+
+* Fill in the required information
+
+    > As soon as the desired asset is selected, the app computes the asset's fingerprint and opens REGISTER PROPERTY RIGHTS screen which allows users to provide more detailed information for the asset.
```

Comment:
"the REGISTER PROPERTY RIGHTS screen"

", which allows"

Probably "about the asset"